eZ/Publish/API/Repository/Tests/SearchServiceTest.php

```diff
@@ -4499,4 +4499,95 @@ private function getContentInfoFixtureClosure($closure = null)
             }
         };
     }
+
+    /**
+     * Test for the findContent() method.
+     *
+     * @see \eZ\Publish\API\Repository\SearchService::findContent()
+     */
+    public function testNonImplementedLegacySearchNonBlock()
```

Comment:
I honestly have no idea what that means :)
maybe `testFieldCriterionForContentsWithIdenticalFieldIdentifiers`
and elaborate a bit more in PhpDoc:
```
Test searching using Field Criterion where the given Field Identifier exists in both searchable and non-searchable Fields.
```